src/services/destination/postTransformation.ts

```diff
@@ -0,0 +1,156 @@
+import cloneDeep from 'lodash/cloneDeep';
```

Comment:
This file looks a little too bulky to me. I mean I felt hard to read. If we can split this into something like below
```typescript
abstract class AbstractPostTransformationHandler {
 static success(...args: any[]): any | any[];
 static failure(...args: any[]): any;
}

// src/services/destination/postTransformation/processor.ts
class ProcessorPostTransformationHandler extends  AbstractPostTransformationHandler{
 success(a, b, c) {
  // your logic
 }
 failure(error, d, e) {
  // your logic
 }
}
```
Similarly we can move each of the success, failure handlers to their specific files.
The goal is to make the file more readable and easily understandable.

Let me know your thoughts

Reply:
If making this change is a concern wrt time or something we can do this later as well. I feel this'd be much better approach & if we can do this now, it'll be nice 🙂